android/src/test/java/com/amplitude/android/plugins/AndroidLifecyclePluginTest.kt

```diff
@@ -496,4 +496,30 @@ class AndroidLifecyclePluginTest {
         verify(exactly = 0) { mockedPlugin.track(capture(tracks)) }
         Assertions.assertEquals(0, tracks.count())
     }
+
+    @Test
+    fun `test deep link opened event is not tracked when URL is missing`() = runTest {
+        setDispatcher(testScheduler)
+        configuration.defaultTracking.deepLinks = false
```

Comment:
should we enable DET here?

Reply:
Yes, good catch! Updated to `true` now. Thanks for reviewing!